templates/builders.go

```diff
@@ -0,0 +1,8 @@
+package templates
+
+import "embed"
+
+var (
+	//go:embed all:builders
```

Comment:
Maybe this is a good time for a rename from builders to Languages or Dockerfiles? Builders is left over from `draft-classic`

Reply:
that's a great idea I was wondering why it was builders haha